components/Shipping/basic_usage.rst

```diff
@@ -125,14 +125,14 @@ In all examples is used an exemplary class implementing **ShippableInterface**,
 Shipping Category
 -----------------
 
-Every shipping category has two identifiers, an ID and name. You can access those by calling ``->getId()`` and ``->getName()``
-methods respectively. The name is mutable, so you can change it by calling ``->setName('Regular')`` on the shipping category instance.
+Every shipping category has three identifiers, an ID, code and name. You can access those by calling ``->getId()``, ``->getCode()`` and ``->getName()``
+methods respectively. The name and code are mutable, so you can change them by calling ``->setCode('X12XXYW')`` and ``->setName('Regular')`` on the shipping category instance.
```

Comment:
Despite the fact that we have a setter the `code` should not be changed after entity is created, so I would ommit saying that `you can change it` :)